rax-cdn.rst

```diff
@@ -0,0 +1,230 @@
+=========================
+ Rackspace CDN using Heat
+=========================
+
+Brief summary
+=============
+
+Rackspace CDN gives you the power to accelerate content on any public resource at
+Rackspace. It provides a simple API and Control Panel experience for you to manage your
+CDN-enabled domains and the origins and assets associated with those domains. You can use
+Rackspace Cloud Orchestration to simplify content deployment and CDN configuration.
+
+Note that Rackspace CDN is a separate service and is different from Rackspace Cloud Files
```

Comment:
I don't want to confuse the two any more than they already might, so I'm going to leave the links as Rackspace CDN only.